app/src/androidTest/java/com/conupods/OutdoorMaps/View/SearchActivityTest.java

```diff
@@ -0,0 +1,83 @@
+package com.conupods.OutdoorMaps.View;
+
+import com.conupods.OutdoorMaps.View.SearchView.SearchActivity;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import androidx.test.platform.app.InstrumentationRegistry;
+import androidx.test.rule.ActivityTestRule;
+import androidx.test.uiautomator.UiDevice;
+import androidx.test.uiautomator.UiObject;
+import androidx.test.uiautomator.UiObjectNotFoundException;
+import androidx.test.uiautomator.UiSelector;
+
+import static junit.framework.TestCase.fail;
+import static org.junit.Assert.assertNotNull;
+
+public class SearchActivityTest {
+    private final static String TAG = "SEARCH_ACTIVITY_TEST";
+    private UiDevice mDevice =  UiDevice.getInstance(InstrumentationRegistry.getInstrumentation());
+    //private ModeSelect modeSelectActivity = null;
+
+
+    @Rule
+    public ActivityTestRule<SearchActivity> searchActivityRule =
+            new ActivityTestRule<>(SearchActivity.class);
+
+    //TODO, When ModeSelect View is pushed, uncomment to test transition from searchView
+   /** @Rule
+    public ActivityTestRule<ModeSelectActivity> modeSelectActivityRule =
+            new ActivityTestRule<>(ModeSelectActivity.class);
+*/
+    @Before
+    public void setUp() {
+
```

Comment:
if u are not using setup delete this 
also see my updates on discord since one test was failing